.ruff.toml

```diff
@@ -13,14 +17,16 @@ exclude = [
     "include"
 ]
 dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
-target-version = "py310"
 
-[per-file-ignores]
+[lint.per-file-ignores]
 "connectors/*" = ["S608"]
 "scripts/verify.py" = [ "EM" ]
 "setup.py" = [ "EM" ]
-"tests/*" = ["B017", "S101", "S"]
+"tests/*" = ["B017", "S101", "S", "ASYNC110"]
 "tests/conftest.py" = [ "EM" ]
 
-[isort]
+[lint.isort]
 known-first-party=["connectors", "tests"]
+
+[lint.flake8-copyright]
+notice-rgx = "#\n# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one\n# or more contributor license agreements. Licensed under the Elastic License 2.0;\n# you may not use this file except in compliance with the Elastic License 2.0.\n#"
```

Comment:
That's pretty much our header, this config verifies that first 4096 bytes of the file start with this regex

Reply:
omg this is amazing